class.two-factor-force.php

```diff
@@ -373,12 +378,13 @@ public static function global_force_2fa_field() {
 	public static function global_force_2fa_by_role_field() {
 		$forced_roles          = self::get_forced_user_roles();
 		$is_universally_forced = self::get_universally_forced_option();
+		$roles                 = array_merge( [ 'super-admin' => [ 'name' => 'Super Administrator' ] ], get_editable_roles() );
```

Comment:
Does this need to be translatable? There's a core string for 'Super Admin' so I'd use that I think.
```suggestion
		$roles                 = array_merge( [ 'super-admin' => [ 'name' => __( 'Super Admin' ) ] ], get_editable_roles() );
```